README.md

```diff
@@ -62,7 +62,7 @@ OPTIONS:
 
 The `--threshold` flag accepts [multiple representations](https://docs.rs/byte-unit/4.0.12/byte_unit/struct.Byte.html#examples-2), like `10 GB`, `10 GiB`, or `10GB`. On Linux, percentage-based thresholds like `50%` are also supported.
 
-The `--min-age` flag accepts [multiple representations](https://docs.rs/parse_duration/2.1.1/parse_duration/), such as `4 days` or `1 hour`.
+The `--min-age` flag accepts [multiple representations](https://docs.rs/humantime/latest/humantime/fn.parse_duration.html), such as `4days` or `1hour`.
```

Comment:
If humantime accepts `4 days` or `1 hour`, I prefer to keep the space in the examples. Or maybe we could add some variation, like:

```
such as `4d` or `1 hour`.
```